tools/docker-compose/ansible/roles/sources/templates/docker-compose.yml.j2

```diff
@@ -220,6 +220,19 @@ services:
     privileged: true
   {% endfor %}
 {% endif %}
+{% if enable_vault|bool %}
+  vault:
+    image: vault:latest
+    container_name: tools_vault_1
+    hostname: vault
+    ports:
+      - "1234:1234"
+    environment:
+      VAULT_DEV_ROOT_TOKEN_ID: "myroot"
```

Comment:
Hardcoded credentials should be avoided, even in the dev env. We have a pattern for this:

https://github.com/ansible/awx/blob/1f939aa25e4ca32fa8a37a469ac929fc73ff9be3/tools/docker-compose/ansible/roles/sources/tasks/main.yml#L12-L37